compiler/luci-interpreter/src/kernels/Utils.h

```diff
@@ -39,6 +39,16 @@ inline int32_t computePadding(int32_t stride, int32_t dilation_rate, int32_t in_
   return padding > 0 ? padding : 0;
 }
 
+inline int32_t computePaddingWithOffset(int stride, int dilation_rate, int in_size, int filter_size,
```

Comment:
Please use int32_t for dimension values (for consistency).

Reply:
Okay, I will fix it!